packages/@aws-cdk/aws-kinesisfirehose-destinations/lib/s3-bucket.ts

```diff
@@ -20,6 +20,26 @@ export interface S3BucketProps extends CommonDestinationProps {
    * @default - UNCOMPRESSED
    */
   readonly compression?: Compression;
+
+  /**
+   * A prefix that Kinesis Data Firehose evaluates and adds to failed records before writing them to S3.
+   *
+   * This prefix appears immediately following the bucket name.
+   * @see https://docs.aws.amazon.com/firehose/latest/dev/s3-prefixes.html
+   *
+   * @default "YYYY/MM/DD/HH"
+   */
+  readonly errorOutputPrefix?: string;
+
+  /**
+   * A prefix that Kinesis Data Firehose evaluates and adds to records before writing them to S3.
+   *
+   * This prefix appears immediately following the bucket name.
+   * @see https://docs.aws.amazon.com/firehose/latest/dev/s3-prefixes.html
+   *
+   * @default "YYYY/MM/DD/HH"
+   */
+  readonly prefix?: string;
```

Comment:
How about a more specific name, like `dataOutputPrefix`, to align it more closely with `errorOutputPrefix`?

I'd rather avoid the generic name `prefix` if we have a different property also called `prefix` in the same interface.